### PR TITLE
Divide contacts section into two

### PR DESCRIPTION
**Pull request by KEREMADALI**

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -4,15 +4,15 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous"><!-- Begin Jekyll SEO tag v2.7.1 -->
-<meta name="generator" content="Jekyll v4.2.0" />
+<meta name="generator" content="Jekyll v4.3.1" />
```
Comment (savas-guven) [inline 404.html]: why was this needed?

**Review by savas-guven — APPROVED** [review]

**Review by detaybey — APPROVED** [review]

File: 404.html (hunk 1 above)
Comment (KEREMADALI) [reply 404.html]: I have just downloaded the latest version it was not a neccesity
